templates/docs/api.html

```diff
@@ -315,27 +398,174 @@ <h3 id="virsh-virsh-virtual-systems-1">virsh (Virsh (virtual systems))</h3>
       </div>
     </main>
   </div>
+{% endblock %}
+{% block js_extra %}
   <script src="/static/js/highlight.pack.js"></script>
   <script>
-    // Set up highlightjs
-    hljs.initHighlightingOnLoad();
```

Comment:
We probably want to keep this as it enables syntax highlighting in `code` blocks.